patches/minecraft/net/minecraft/command/EntitySelector.java.patch

```diff
@@ -12,3 +12,30 @@
      private static final Set<String> field_179666_d = Sets.newHashSet(new String[] {"x", "y", "z", "dx", "dy", "dz", "rm", "r"});
  
      @Nullable
+@@ -78,6 +78,8 @@
+     }
+ 
+     public static <T extends Entity> List<T> func_179656_b(ICommandSender p_179656_0_, String p_179656_1_, Class <? extends T > p_179656_2_)
++    {return net.minecraftforge.common.ForgeHooks.entitySelectorProxy.matchEntities(p_179656_0_, p_179656_1_, p_179656_2_);}
++    public static <T extends Entity> List<T> matchEntitiesDefault(ICommandSender p_179656_0_, String p_179656_1_, Class <? extends T > p_179656_2_)
+     {
+         Matcher matcher = field_82389_a.matcher(p_179656_1_);
+ 
+@@ -693,6 +695,8 @@
+     }
+ 
+     public static boolean func_82377_a(String p_82377_0_)
++    {return net.minecraftforge.common.ForgeHooks.entitySelectorProxy.matchesMultiplePlayers(p_82377_0_);}
```

Comment:
you dont need to compact it THIS much brances on new lines is fine.
The patches just need to be clear what they are doing at a quick glance.